include/deal.II/fe/mapping_fe_field.h

```diff
@@ -62,50 +62,91 @@ DEAL_II_NAMESPACE_OPEN
  *    dhq.distribute_dofs(fesystem);
  *    Vector<double> eulerq(dhq.n_dofs());
  *    const ComponentMask mask(spacedim, true);
- *    MappingFE<dim,spacedim> map(eulerq, dhq, mask);
+ *    MappingFEField<dim,spacedim> map(eulerq, dhq, mask);
  *    map.update_euler_vector_using_triangulation(eulerq);
```

Comment:
I still don't understand what the last line does. Why do you have to pass the vector in twice?